zokrates_core/src/helpers/mod.rs

```diff
@@ -67,7 +67,7 @@ impl Helper {
     }
 
     pub fn bits() -> Self {
-        Helper::Wasm(WasmHelper::from(WasmHelper::BITS_WASM))
+        Helper::Rust(RustHelper::Bits)
```

Comment:
Do we have some docs/notes on the current approach to helpers? Do we want to remove the WASM Helpers from the codebase?

Reply:
Yes I think we should remove them altogether but it can be a separate PR. In this one we would merely stop using wasm for unpacking